geoh5py/groups/property_group.py

```diff
@@ -156,6 +198,22 @@ def property_group_type(self) -> str:
     def property_group_type(self, group_type: str):
         self._property_group_type = group_type
 
+    def remove_properties(self, data: Data | list[Data]):
```

Comment:
what if we want to remove property from a property name or uid?

Reply:
We could allow for uuid, true. Maybe not from `name` though since too ambiguous.